vip-cache-manager/vip-cache-manager.php

```diff
@@ -556,6 +559,23 @@ public function queue_old_permalink_purge( $post_ID, $post_after, $post_before )
 		}
 	}
 
+	protected function normalize_purge_url( $url ) {
+		$url = esc_url_raw( $url );
+
+		$parsed_url = parse_url( $url );
+
+		// Strip off any URL fragments (i.e. `#search=xyz`) since that is ignored by the cache
```

Comment:
As our caching purges all query string variants of a URL, should we also strip the `query` portion of the URL? This might further reduce the number of URLs, but does make the system less DRY as a whole in that we would need to change the code here and in the cache config if we needed to remove/adjust this behaviour.

Reply:
Is that part of our Varnish config, or Mangle?

Reply:
I think I've seen this in the Varnish config, and certainly the behaviour existed before Mangle was implemented.

Reply:
Confirmed, it's in the Varnish config.

Reply:
I think it's okay to leave the `query` bits in the PURGE URLs for now. Can revisit later if we find PURGE counts are still high for sites.